src/core/Microsoft.PowerToys.Settings.UI.Lib/ImageSize.cs

```diff
@@ -141,8 +141,8 @@ public double Width
 
             set
             {
-                int newWidth = -1;
-                int.TryParse(value + string.Empty, out newWidth);
+                double newWidth = -1;
+                double.TryParse(value + string.Empty, out newWidth);
```

Comment:
I would use value.ToString() instead of concatenating it with empty string, same below

Reply:
I changed it such that it no longer converts to string, since `value` is already double. It seems like the `int.TryParse(value+string.Empty)` logic was added for the case where the NumberBox is empty, value becomes NaN. I replaced the string logic to a check for NaN instead.